packages/lib/string-utils.ts

```diff
@@ -2,7 +2,7 @@ const Entities = require('html-entities').AllHtmlEntities;
 const htmlentities = new Entities().encode;
 const stringUtilsCommon = require('./string-utils-common.js');
 
-export const pregQuote = stringUtilsCommon.pregQuote;
+export const pregQuote = stringUtilsCommon.pregQuote as (str: string, delimiter?: string)=> string;
```

Comment:
`pregQuote` is imported from `string-utils-common.js`. As such, it previously had no type information. This line causes `SearchEngine.queryTermToRegex` to return `string` instead of `any`.